client/ayon_motionbuilder/api/testing/tests.py

```diff
@@ -0,0 +1,74 @@
+import contextlib
+import pyblish.util
+from pyfbsdk import FBSystem
+from ayon_core.pipeline import (
+    registered_host
+)
+from ayon_core.pipeline.create import CreateContext
+
+from . import lib
+
+# Needed for transition phase for asset/subset renaming. Can be hardcoded once
+# transition is done.
+product_key_name = "productName"
+product_type_key_name = "productType"
```

Comment:
This isn't relevant anymore - since `productName` and `productType` is already the new naming and ayon-core aligns with?